components/Toast.js

```diff
@@ -95,12 +118,17 @@ const getIcon = toast => {
   }
 };
 
-const Toast = ({ toast, timeLeft, onClose }) => {
+const Toast = ({ toast, timeLeft, onClose, variant }) => {
   const intl = useIntl();
   const [isClosing, setClosing] = React.useState(false);
   const color = getColor(toast);
+  const variantStyle = getVariant(variant);
```

Comment:
The implementation should rely more on styled-components, having this `getVariant` helper actually defeats the purpose of using a styled-system variant. I'll push an update today.